core/app/models/spree/pricers/conservative_line_item_pricer.rb

```diff
@@ -0,0 +1,23 @@
+module Spree
+  module Pricers
+    class ConservativeLineItemPricer
+      class VariantMissing < StandardError; end
+      class CurrencyMissing < StandardError; end
+
+      attr_reader :line_item
+
+      def initialize(line_item)
+        @line_item = line_item
+      end
+
+      def price
```

Comment:
Could we get some YARD doc for these methods, and the class?

For a method called `price` I would expect it to return a `Spree::Price` but instead it returns a BigDecimal. I would like to avoid returning `BigDecimal` in any of these scenarios. I feel it makes more sense to return a Money object or Spree::Price.